src/compat/event_listeners.ts

```diff
@@ -192,6 +196,35 @@ function isInBackground$() : Observable<boolean> {
     .pipe(startWith(false));
 }
 
+/**
+ * @returns {Observable}
+ */
+function videoIsVisible$(
+  mediaElement: HTMLMediaElement
+) : Observable<boolean> {
+  const isPIPActiveAtInit = (document as any).pictureInPictureElement != null &&
+    (document as any).pictureInPictureElement === mediaElement;
+
+  const visibilityChange$ = visibilityChange().pipe(map((x) => !!x));
```

Comment:
We might want to generalize that as `isVisible$` as it makes more sense and update the code for ``isInBackground$``